ddtrace/tracer/option.go

```diff
@@ -359,23 +368,29 @@ func (c *config) loadAgentFeatures() {
 		Endpoints     []string `json:"endpoints"`
 		ClientDropP0s bool     `json:"client_drop_p0s"`
 		StatsdPort    int      `json:"statsd_port"`
+		FeatureFlags  []string `json:"feature_flags"`
 	}
 	var info infoResponse
 	if err := json.NewDecoder(resp.Body).Decode(&info); err != nil {
 		log.Error("Decoding features: %v", err)
 		return
 	}
-	c.features.DropP0s = info.ClientDropP0s
-	c.features.StatsdPort = info.StatsdPort
+	c.agent.DropP0s = info.ClientDropP0s
+	c.agent.StatsdPort = info.StatsdPort
 	for _, endpoint := range info.Endpoints {
 		switch endpoint {
 		case "/v0.6/stats":
-			if c.HasFeature("discovery") {
-				// client-stats computation is off by default
-				c.features.Stats = true
+			// disable_stats feature flag is here temporarily to allow reverting
+			// in case of any issues.
+			if !c.HasFeature("disable_stats") {
+				c.agent.Stats = true
```

Comment:
We need to prove this feature out with real production customers before turning it on by default for everyone.

Reply:
Done in https://github.com/DataDog/dd-trace-go/pull/1069/commits/719c849101c8a09066fc782ff544a67a52485d33. Thanks for jumping in.